src/tests/ftest/deployment/disk_failure.yaml

```diff
@@ -0,0 +1,60 @@
+hosts:
+  test_servers: 4
+  test_clients: 1
+timeout: 420
+daos_server:
+  pattern_timeout: 60
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["0000:00:00.0"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+      targets: 16
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      bdev_class: nvme
+      bdev_list: ["0000:00:00.1"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem1"]
+      scm_mount: /mnt/daos1
+      targets: 16
+pool:
+  size: 80%
+  control_method: dmg
+  svcn: 3
+container:
+  type: POSIX
+  control_method: daos
+  oclass: RP_2GX
+  properties: "cksum:crc16,rf:1"
+ior:
+  client_processes:
+    np: 2
```

Comment:
Any reason to not use more processes? The IOR block size would need to be reduced, but using more processes would create more stress and probably be faster

Reply:
We don't want things to be fast here. We want the IO process to take more time so we can try to evict targets (more target in future testing).